sources/api/types.go

```diff
@@ -57,19 +57,27 @@ func (a *AggregateData) Merge(b *AggregateData) {
 	a.Events = append(a.Events, b.Events...)
 }
 
+type ContainerSpec struct {
+	cadvisor.ContainerSpec
+	CpuRequest    int64
```

Comment:
That makes sense. We should handle both request and limit though. Values from cAdvisor will translate to limits, but that can be overridden in the case of Kubernetes.